pkg/azuredisk/azuredisk.go

```diff
@@ -321,6 +321,9 @@ func (d *Driver) CreateManagedDisk(ctx context.Context, options *ManagedDiskOpti
 
 	_, err = d.cloud.DisksClient.CreateOrUpdate(ctx, options.ResourceGroup, options.DiskName, model)
 	if err != nil {
+		if strings.Contains(err.Error(), "NotFound") {
+			return "", status.Errorf(codes.NotFound, "the specify snapshot(%s) is not found under rg(%s)", options.SourceResourceID, options.ResourceGroup)
```

Comment:
pls also have the error msg in returned error

Reply:
done.